pettingzoo/sisl/waterworld/waterworld_models.py

```diff
@@ -135,8 +136,8 @@ def __init__(
     @property
     def observation_space(self):
         return spaces.Box(
-            low=np.float32(-2 * np.sqrt(2)),
-            high=np.float32(2 * np.sqrt(2)),
+            low=np.float32(-4 * np.sqrt(2)),
+            high=np.float32(4 * np.sqrt(2)),
```

Comment:
Just curious, what are these magic numbers?

Reply:
They should be $\pm\sqrt{2}$, the max speed is defined for $v_{x}$ and $v_y$, so the max magnitude for the speed $v = [v_x, v_y]$ would be $\sqrt{v_x^2 + v_y^2} = \sqrt{2}v_\mathrm{max}$